point_of_sale/advanced_pricing_features/pricelists.rst

```diff
@@ -0,0 +1,46 @@
+=================================
+Using Pricelists in Point of Sale
+=================================
+
+You probably know the concept of happy hour. During a certain period of
+time, the barman gives a discount on some drinks (usually 50% off or a
+buy one get one free). When the period is over, everything goes back to
+normal. But what is the link with Odoo?
+
+In Odoo, you can set up happy hours. It’s one of the many possible uses
+of *Pricelists*. Those *Pricelists* will allow the creation of
+multiple prices for the same product: a regular one and a special one
+for happy hours. Available in the *PoS* app, those are really
+convenient.
+
+Set up Pricelists
+=================
+
+To set up a *Pricelist*, go to :menuselection:`Point of Sale --> Configuration --> Configuration` 
+and enable the *Pricelist* feature. Then, go to :menuselection:`Point of Sale --> Configuration 
+--> Point of Sale` and enable the *Pricelist* for the *PoS*.
+
+.. image:: media/pricelists_01.png
+   :align: center
+
+Now, you can create *Pricelists* by clicking on the *Pricelists* link. 
+Then, set it up by choosing the product category you want to include in your happy hour 
+and the discount.
```

Comment:
```suggestion
Now, click on *Pricelists* and choose the product category you want to include in your happy hour
and the discount.
```

Reply:
I do prefer keeping it the way I wrote it :)